### PR TITLE
fixed rand init error message

### PR DESCRIPTION
**Pull request by ChillerDragon**
the ``dbg_msg()`` func doesn't work until the
``IEngine *pEngine = CreateEngine("DDNet", Silent);`` line gets executed.

I used a bool to leave the code order untouched c:

### CODE REVIEW

File: src/engine/client/client.cpp [hunk 1]
```diff
@@ -3447,8 +3448,7 @@ int main(int argc, const char **argv) // ignore_convention
 
 	if(secure_random_init() != 0)
 	{
-		dbg_msg("secure", "could not initialize secure RNG");
-		return -1;
+		IsRandFail = true;
```
Comment (msiglreith) [inline src/engine/client/client.cpp]: shorter: `bool IsRandFail = secure_random_init() != 0;`

File: src/engine/client/client.cpp [hunk 2]
```diff
@@ -3467,6 +3467,12 @@ int main(int argc, const char **argv) // ignore_convention
 	IEngineMap *pEngineMap = CreateEngineMap();
 	IEngineMasterServer *pEngineMasterServer = CreateEngineMasterServer();
 
+	if (IsRandFail)
```
Comment (msiglreith) [inline src/engine/client/client.cpp]: why not directly after engine creation?

**Comment by heinrich5991** [comment]
Just out of interest, did you encounter this, @ChillerDragon?

**Comment by ChillerDragon** [comment]
@heinrich5991 no. but i tested it by putting a ``|| true`` in the if statement and nothing happend.
@msiglreith idk how GitHub works and im not sure if you can see my response.
1. i responded to your first comment: "ikr... i thought about it but it. Also to use a int to fetch the error value but im shy with the fancy stuff cuz im new/noob dev."

2. and to your second comment i responded "dosen't look too good imo."

File: src/engine/client/client.cpp (hunk 2 above)
Comment (heinrich5991) [reply src/engine/client/client.cpp]: No space between `if` and `(` please. While you're at it, you could rename the variable to something like `DidRandInitFail` or `RandInitFailed` or similar.

**Pull request merged**